liquidator/liquidator.js

```diff
@@ -214,7 +218,9 @@ class Liquidator {
         lockedCollateral: receipt.events.LiquidationCreated.returnValues.lockedCollateral,
         liquidatedCollateral: receipt.events.LiquidationCreated.returnValues.liquidatedCollateral
       };
-      this.logger.info({
+
+      // This log level can be overridden by specifying `positionLiquidated` in the `logOverrides`. Otherwise, use info.
+      this.logger[this.logOverrides.positionLiquidated ? this.logOverrides.positionLiquidated : "info"]({
```

Comment:
this is the only change needed to apply the override. If there is an override for the exact key named `positionLiquidated` then it will be applied to the log that is committed. if that key is not present then use the existing `info` log level`. What's nice about this is that the log levels are validated in the `createObjectFromDefaultProps` call so there is no chance this will break the behavior.